app/frontend/components/pages/delegations/delegationHistoryPage.tsx

```diff
@@ -0,0 +1,145 @@
+import {h, Component, Fragment} from 'preact'
+import actions from '../../../actions'
+import {connect} from '../../../libs/unistore/preact'
+import {CopyPoolId} from './common'
+import toLocalDate from '../../../../frontend/helpers/toLocalDate'
+
+export enum DelegetionHistoryItemType {
+  StakeDelegation,
+  StakingReward,
+  RewardWithdrawal,
+}
+
+export interface DelegetionHistoryObject {
+  type: DelegetionHistoryItemType
+  epoch: number
+  dateTime: Date
+}
+
+export interface StakePool {
+  id: string
+  name: string
+}
+
+export interface StakeDelegation extends DelegetionHistoryObject {
+  newStakePool: StakePool
+  oldStakePool?: StakePool
+}
+
+const EpochDateTime = ({epoch, dateTime}: {epoch: number; dateTime: Date}) => {
+  return (
+    <div className="grey margin-bottom">
+      Epoch {epoch}, {toLocalDate(dateTime)}
+    </div>
+  )
+}
+
+const StakeDelegationItem = ({stakeDelegation}: {stakeDelegation: StakeDelegation}) => {
+  return (
+    <li className="delegations-history-item">
+      <div className="label">Stake Delegation</div>
+      <EpochDateTime epoch={stakeDelegation.epoch} dateTime={stakeDelegation.dateTime} />
+      <div>
+        New pool: <span className="bold">{stakeDelegation.newStakePool.name}</span>
+        <CopyPoolId value={stakeDelegation.newStakePool.id} />
+      </div>
+      {stakeDelegation.oldStakePool ? (
+        <div>
+          Previous pool: {stakeDelegation.oldStakePool.name}
+          <CopyPoolId value={stakeDelegation.oldStakePool.id} />
+        </div>
+      ) : (
+        ''
+      )}
+    </li>
+  )
+}
+
+export interface StakingReward extends DelegetionHistoryObject {
+  reward: number
+  stakePool: StakePool
+}
+
+const StakingRewardItem = ({stakingReward}: {stakingReward: StakingReward}) => {
+  return (
+    <li className="delegations-history-item">
+      <div>
+        <div className="label">Staking reward</div>
+        <EpochDateTime epoch={stakingReward.epoch} dateTime={stakingReward.dateTime} />
+      </div>
+      <div>
+        <div>Reward: {stakingReward.reward}</div>
+        <div className="grey">
+          {stakingReward.stakePool.name}
+          <CopyPoolId value={stakingReward.stakePool.id} />
+        </div>
+      </div>
+    </li>
+  )
+}
+
+export interface RewardWithdrawal extends DelegetionHistoryObject {
+  credit: number
+}
+
+const RewardWithdrawalItem = ({rewardWithdrawal}: {rewardWithdrawal: RewardWithdrawal}) => {
+  return (
+    <li className="delegations-history-item">
+      <div className="space-between">
+        <div>
+          <div className="label">Reward withdrawal</div>
+          <EpochDateTime epoch={rewardWithdrawal.epoch} dateTime={rewardWithdrawal.dateTime} />
+        </div>
+        <div>
+          <div className="transaction-amount credit">{rewardWithdrawal.credit}</div>
```

Comment:
This is the amount in Lovelaces (the smallest amount of ada. 1 lovelace = 0.000001 ada). 
![image](https://user-images.githubusercontent.com/22454337/91537758-1ab7e380-e917-11ea-9f93-5d49ba29c40a.png)

We should show this in ADA, not in lovelace. Check the file _app/frontend/helpers/adaConverters.ts_ and the function _toAda_. I believe stakingReward.reward will be the same case

Reply:
Fixed